Src/BootCamp.Chapter/Checks.cs

```diff
@@ -14,28 +15,67 @@ namespace BootCamp.Chapter
     /// </summary>
     public static class Checks
     {
+        public static string Name { get; set; }
+        public static string Surname { get; set; }
+        public static int Age { get; set; }
+        public static int Weight { get; set; }
+        public static float Height { get; set; }
+
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            Console.WriteLine(message);
+            if (!int.TryParse(Console.ReadLine(), out int age))
+            {
+                //Console.WriteLine("Age Should be a Number");
+            }
+            else
+            {
+                Age = age;
+            }
+
+            
+            if (!int.TryParse(Console.ReadLine(), out int weight))
+            {
+                //Console.WriteLine("Weight must be a Number");
+            }
+            else
+            {
+                Weight = weight;
+            }
+
+            return Age;
         }
 
         public static string PromptString(string message)
         {
-            // To do: call your implementation. 
-            return "";
+            Console.WriteLine(message);
+            
```

Comment:
Please refactor the function not to mess with the program state- not a global state as well (static!). This means, that a function takes input and returns output. It should rarely change state of something static. This is justified for object methods (which we will learn in chapter 2), so for now avoid changing global values in functions.

```cs
public static string PromptString(string message)
{
  // ask for input
  Console.WriteLine(message);
  // return input (+ convert it if needed)
  return Console.ReadLine();
}
```

Also, due to the scope of a function being small, adding a blank line between every line of code is not so good. It's very related.